_posts/en/newsletters/2020-01-08-newsletter.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Bitcoin Optech Newsletter #79'
+permalink: /en/newsletters/2020/01/08/
+name: 2020-01-08-newsletter
+slug: 2020-01-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes the final week of the organized
+taproot review, describes a discussion about coinjoin mixing without
+either equal value inputs or outputs, and mentions a proposal to encode
+output script descriptors in end-user interfaces.  Also included is our
+regular section about notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Final week of organized taproot review:** December 17th was the
+  final scheduled [meeting][taproot meeting] of the taproot review
+  group.  Pieter Wuille posted the [slides][wuille slides] from a
+  presentation he gave summarizing progress, including text indicating
+  he though the proposal was "nearly ready".  Wuille also proposed a
+  [minor change][wuille suggestion] to the tapleaf versioning.  Also
+  briefly mentioned was the [discussion][zmn post] started by ZmnSCPxj
+  on the Lightning-Dev mailing list about how precisely taproot could be
+  integrated with LN to provide improved privacy and scalability.
+
+- **Coinjoins without equal value inputs or outputs:** Adam Ficsor
+  (nopara73) started a [discussion][ficsor non-equal] on the Bitcoin-Dev
+  mailing list about two previously-published papers ([1][cashfusion],
+  [2][knapsack]) describing coinjoins that didn't use either equal-value
+  inputs or outputs.  Previous attempts at non-equal mixes were [easy to
+  compromise][coinjoin sudoku], but if an improved method was found, it
+  could significantly improve the privacy of coinjoins by making their
+  transactions look like [payment batching][].  This seemed especially
+  relevant after reports that a popular exchange was investigating users
+  participating in the chaumian-style coinjoins created by Wasabi
+  Wallet.  Although several ideas were discussed, we think the summary
+  by Lucas Ontivero captures the essence of the overall conclusion: "In
```

Comment:
perhaps link to Ontivero's post (https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-December/017544.html)?

Reply:
added